frontend/src/pages/home/index.jsx

```diff
@@ -123,22 +46,54 @@ const Dashboard = (props) => {
     return <CircularProgress />;
   }
 
-  const fixedHeightPaper = clsx(classes.paper, classes.fixedHeight);
+  const data = ({ color }) => ({
+    datasets: [
+      {
+        data: [27, 73],
+        backgroundColor: [
+          'rgb(0, 0, 0, 0)',
+          color,
+        ]
+      },
+    ],
+  });
+
+  const options = ({ color }) => ({
+    cutoutPercentage: 75,
+    tooltips: { enabled: false },
+    hover: { mode: null },
+    responsive: true,
+    elements: {
+      center: {
+        text: '73%',
+        color: color || '#FFFFFF',
+      }
+    }
+  });
+
   return (
     <PageContainer>
       <Grid container spacing={3}>
-        {/* Recent Deposits */}
-        {classList.map((aClass) => (
-          <ClassBox
-            className={fixedHeightPaper}
-            class={aClass}
-            key={aClass.id}
-          />
-        ))}
-        {/* Chart */}
-        <Grid item xs={12} md={12} lg={12}>
-          <Paper className={fixedHeightPaper}>
-            <Chart />
+        {/* Widgets with general information */}
+        <Widget left={6} right="lessen" color={theme.widget.primary.main}></Widget>
+        <Widget left={104} right="leerlingen" color={theme.widget.secondary.main}></Widget>
+        <Widget left={25} right="na te kijken opdrachten" color={theme.widget.tertiary.main}></Widget>
+        <Grid item xs={12} md={4}>
+          <Paper className={classes.paper}>
```

Comment:
kunnen we van deze card met de donut grafiek erin niet leuk een apart component maken? die een title, data, en options meekrijgt ofzo?want nu heb je veel duplicate code